README.rst

```diff
@@ -1,20 +1,18 @@
-===========  =================================================================================================================================
-Info         NI VeriStand Python Support
+===========  ====================================================
+Info         VeriStand Python Support
 Author       National Instruments
-===========  =================================================================================================================================
-
-.. _about_section:
+===========  ====================================================
 
 About
 =====
-The **niveristand** package contains an API (Application Programming Interface) that interacts with NI VeriStand systems.
+The **niveristand** package contains an API (Application Programming Interface) that interacts with VeriStand systems.
 The package is implemented in Python. NI created and supports this package.
 
 Requirements
 ============
-**niveristand** |version| has the following requirements:
+**niveristand** 0.9.x has the following requirements:
 
-* NI VeriStand 2018 running on the same system as the Python code.
+* VeriStand 2018+ running on the same system as the Python code.
```

Comment:
[optional] VeriStand 2018 or later

Reply:
I'll bring it up with the tech writer and just leave it for later.